configs/14.0/release_notes/relfixes.html

```diff
@@ -1,16 +1,83 @@
+<li>POWER10 enablement.</li>
 <li>POWER9 enablement.</li>
 <li>POWER9 optimized scheduler.</li>
 <li>Requires at least POWER8 for ppc64le.</li>
 <li>POWER8 Transactional Memory enablement.</li>
 <li>POWER8 Crypto Operations enablement.</li>
-<li>POWER8 and POWER9 optimized system libraries.</li>
+<li>POWER8, POWER9 and POWER10 optimized system libraries.</li>
 <li>POWER9 _Float128 enablement as specified by ISO/IEC TS 18661-3.</li>
 <li>GCC creates binaries using --mcpu=power8 --mtune=power9 by default.</li>
 
 <h3>Advance Toolchain new location</h3>
 <p>Since the end of August 2019, Advance Toolchain has been available from a new FTP server: <a href="ftp://public.dhe.ibm.com/">public.dhe.ibm.com</a></p>
 <p>Please be advised the FTP server at Unicamp won't be updated any longer.</p>
 
+<h3>New features in 14.0-0</h3>
+<ul>
+  <li>Support for Ubuntu 20.04.</li>
+  <li>GCC provides:</li>
+  <ul>
+    <li>Version 10.2, see the <a href="https://gcc.gnu.org/gcc-10/changes.html">release notes</a> for details.</li>
+    <li>MMA support.</li>
```

Comment:
Do we want to provide some context for "MMA"? Is there a link for more information?

Reply:
Added a link to GCC documentation.